Data/models/Bookmark+Sync.swift

```diff
@@ -15,14 +15,50 @@ extension Bookmark {
     /// Base order is needed to distinguish between bookmarks on different devices and platforms.
     static var baseOrder: String { return Preferences.Sync.baseSyncOrder.value }
     
-    /// Sets order for all bookmarks. Needed after user joins sync group for the first time.
+    public class func isSyncOrderValid(_ value: String) -> Bool {
```

Comment:
Never mind, this doesn't work for a number of reasons.

Reply:
@jhreis though your solution is more readable I would advise to go with Michal's solution as it is more performant (O(n) in this case). @iccub it would be good to have a description of what this function does. Another good to have would be to create a func in String ext to validate syncOrder/versioning

Reply:
we have decided to stay with regex as it catches more weird cases added a comment on it too